test/general/sycl_iterator/sycl_iterator.pass.h

```diff
@@ -0,0 +1,77 @@
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _TEST_SYCL_ITERATOR_PASS_H
```

Comment:
> I think a header should not have ".pass" in its name. It may confuse cmake system and generate an extra target.

Fixed.

Reply:
> What do you think about naming it as "utils_sycl_iterator.h" and moving into "test/support" folder? It may be even better put everything inside "test/support/utils_sycl.h" and remove that header.

I don't think so. It's the header file only for concrete 5 tests. Not more.